core/action.py

```diff
@@ -142,6 +145,12 @@ def process(self, submission, last_good_submission=None):
             message_lines.append("your post appears to be a roo submitted as a text post. All switcharoos should be "
                                  "submitted as link posts for clarity and subreddit organization.")
 
+        if submission_is_NSFW in self.issues:
+                    message_lines.append("your post is linked to a NSFW post. As per the subbreddit rule, this post will not be linked to a roo"
+                                         "Kindly delete the post.")
+                    resubmit = False
+                    action = WARN
```

Comment:
Should this be DELETE instead?

Reply:
Yep, if there isn't any way for them to fix their post, we delete it. If there was just some problem with the link we give them an additional message about resubmitting it along with the delete.

Reply:
Can you change the message to this?

    your post is linked to a NSFW post. As per r/switcharoo house rules, we don't allow submissions from NSFW subreddits. Sorry about that!

Writing the script for the bot is difficult since people don't like being told off by a bot but also don't like fake enthusiasm. So it has to be balanced in-between. I'm open to edits and suggestions if you have any.

Reply:
I'll use your suggestion for this line.